hermes-common/src/main/java/pl/allegro/tech/hermes/common/message/wrapper/MessageMetadata.java

```diff
@@ -38,6 +46,8 @@ public boolean equals(Object obj) {
             return false;
         }
         final MessageMetadata other = (MessageMetadata) obj;
-        return Objects.equals(this.id, other.id) && Objects.equals(this.timestamp, other.timestamp);
+        return Objects.equals(this.id, other.id)
+                && Objects.equals(this.timestamp, other.timestamp)
+                && Objects.equals(this.traceId, other.traceId);
```

Comment:
do we need traceId to identify the object? i think the previous implementation was fine